ckan/tests/legacy/functional/api/test_email_notifications.py

```diff
@@ -409,23 +399,17 @@ def test_01_no_emails_sent_if_turned_off(self):
 
 class TestEmailNotificationsSinceIniSetting(
         mock_mail_server.SmtpServerHarness,
-        pylons_controller.PylonsTestCase):
+        ControllerTestCase):
     '''Tests for the ckan.email_notifications_since config setting.'''
 
     @classmethod
     def setup_class(cls):
-        cls._original_config = config.copy()
 
         # Don't send email notifications for activities older than 1
```

Comment:
I think this comment needs to be moved down a few lines to be with the corresponding code